src/parseInputFile.C

```diff
@@ -3804,6 +3807,9 @@ void EW::processCheckPoint(char* buffer) {
     m_check_point->set_checkpoint_file(filePrefix, cycle, cycleInterval,
                                        bufsize, useHDF5, compressionMode,
                                        compressionPar);
+  if (restartLatestGiven) {
+    m_check_point->set_restart_latest(bufsize);
+  }
   if (restartFileGiven) {
     m_check_point->set_restart_file(restartFileName, bufsize);
   }
```

Comment:
We'd probably want the user to specify either ``restartlatest`` or ``restartfile``, but not both.  So maybe we should add a check that they haven't set both options.

Reply:
I added a check for this.